extensions/PhabBugz/lib/Project.pm

```diff
@@ -267,6 +281,24 @@ sub remove_member {
     push( @{ $self->{remove_members} }, $member_phid );
 }
 
+sub add_project {
+    my ($self, $project) = @_;
+    $self->{add_projects} ||= [];
+    my $project_phid = blessed $project ? $project->phid : $project;
+    $project_phid = get_project_phid($project);
```

Comment:
remove this line as it is redundant and will fail if $project is an object or a PHID.